apps/omg_watcher/test/integration/block_getter_test.exs

```diff
@@ -275,6 +275,83 @@ defmodule OMG.Watcher.Integration.BlockGetterTest do
     JSONRPC2.Servers.HTTP.shutdown(BadChildChainTransaction)
   end
 
+  @tag fixtures: [:watcher_sandbox, :stable_alice, :child_chain, :token, :stable_alice_deposits]
+  test "transaction which is using already spent utxo from exit and happened after m_sv causes to emit invalid_block event",
+       %{stable_alice: alice, stable_alice_deposits: {deposit_blknum, _}} do
+    # TODO remove this tx , use directly deposit_blknum to get_exit_data
```

Comment:
can't this TODO be done easily right away?

Reply:
As I remember there is known issue about exiting/spending directly using utxo from deposit. That's why I left `TODO` + 'deposit_blknum' as var.

Reply:
hm, then maybe no `TODO` at all, except the additional line there really isn't much to improve here, if we spend from deposit directly?

What is the issue?